### PR TITLE
Install Tower

### PR DESCRIPTION
**Pull request by Tompage1994**
Adds ability to install Tower.

I think the roles should probably end up inside of a collection. perhaps something like `redhat_cop.tower_utils` (so a new repo required)?

I'm not sure there will be a good way to run automated tests of this.

Do we think this repo should contain things like this or should it perhaps live elsewhere?

My thoughts where that a set of playbooks would exist under `tower_setup` which would get Tower to a specific configuration (e.g. adding license,some example orgs, some credentials. The kind of stuff that may be considered 'base configuration')

### CODE REVIEW

**Review by aoyawale — APPROVED** [review]
everything looks to me. Yes, we should make it into a collection but this is a good starting point.

File: ansible.cfg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[defaults]
+roles_path=roles
+collections_paths=colelctions
```
Comment (sean-m-sullivan) [inline ansible.cfg]: Typo here on word collections.

**Review by sean-m-sullivan — CHANGES_REQUESTED** [review]
Looks good, just have that one minor typo, I like how you did the 3.6 vs 3.7 rabbitmq changes as well.

File: ansible.cfg (hunk 1 above)
Comment (Tompage1994) [reply ansible.cfg]: Fixed

**Review by sean-m-sullivan — APPROVED** [review]
Looks Good!

**Pull request merged**